ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -68,6 +72,7 @@ public Bundle build() {
 
     public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras) {
         AdConfig adConfig = new AdConfig();
+        adConfig.setMuted(true); // start muted by default.
         if (networkExtras != null) {
             adConfig.setMuted(networkExtras.getBoolean(EXTRA_START_MUTED, false));
```

Comment:
this is old code but if EXTRA_START_MUTED  is not set, wouldnt this overwrite the value from True to False, shouldnt we have default as true instead of false
https://github.com/googleads/googleads-mobile-android-mediation/pull/212#discussion_r385880280

Reply:
fixed.